baldrick/plugins/github_towncrier_changelog.py

```diff
@@ -99,8 +98,7 @@ def verify_pr_number(pr_number, matching_file):
 
 
 def load_towncrier_config(pr_handler):
-    file_content = pr_handler.get_file_contents("pyproject.toml", branch=pr_handler.base_branch)
-    config = loads(file_content)
+    config = pr_handler.get_repo_config()
```

Comment:
Changing from `get_file_contents` to `get_repo_config` seems to break a lot of tests. I don't understand why -- is it some subtlety in how TOML works or I missed something in the mock magic?